src/test/scala/chiselTests/BundleLiteralSpec.scala

```diff
@@ -3,47 +3,25 @@
 package chiselTests
 
 import chisel3._
-import chisel3.core.FixedPoint
-import chisel3.experimental.RawModule
 import chisel3.testers.BasicTester
-import org.scalatest._
+import chisel3.experimental.RawModule
+import chisel3.experimental.BundleLiteralConstructor._
```

Comment:
Suggestion to change `chisel3.experimental.BundleLiteralConstructor` to `chisel3.experimental.BundleLiteral` or `chisel3.experimental.BundleLiterals`

thoughts?

Reply:
I like the idea of `chisel3.experimental.BundleLiterals`